saleor/product/migrations/0130_create_product_description_search_vector.py

```diff
@@ -0,0 +1,93 @@
+# Generated by Django 3.1.5 on 2021-01-21 09:19
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.db import migrations, models
+
+
+def parse_draftjs_content_to_string(definitions):
+    string = ""
+    blocks = definitions.get("blocks")
+    if not blocks or not isinstance(blocks, list):
+        return ""
+    for block in blocks:
+        text = block.get("text")
+        if not text:
+            continue
+        string += f"{text} "
+
+    return string
+
+
+def parse_description_json_field(apps, schema):
+    Product = apps.get_model("product", "Product")
+
+    for product in Product.objects.iterator():
+        product.description_plaintext = parse_draftjs_content_to_string(
+            product.description_json
+        )
+        product.save()
```

Comment:
`save(update_fields=["description_plaintext"])`

Reply:
It's code from 2.11, I would like to keep it the same at it was on 2.11